extensions/wikia/UserLogin/UserLoginHelper.class.php

```diff
@@ -514,7 +514,7 @@ public function showRequestFormConfirmEmail( EmailConfirmation $pageObj ) {
 
 			if ( $user->isEmailConfirmationPending() || !empty( $optionNewEmail ) ) {
 				$out->addWikiMsg( 'usersignup-confirm-email-unconfirmed-emailnotauthenticated' );
-				if ( strtotime( $user->mEmailTokenExpires ) - strtotime( "+6 days 23 hours" ) > 0 )
+				if ( strtotime( $user->getEmailTokenExpires() ) - strtotime( "+6 days 23 hours" ) > 0 )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)